Invalid template in some of the commands to create new Farm projects on the Quick Start page

Hi! 👋 

When trying to create a new Farm project from one of the [npm commands](https://www.farmfe.org/docs/quick-start#create-a-farm-project) (`npm 7+`) available on the [Quick Start page](https://www.farmfe.org/docs/quick-start) of the documentation, I got this error:

```plain
npm create farm@latest my-vue-app -- --template vue

 ⚡ Welcome To Farm !



 error: failed to parse 'vue': vue is not a valid template. Valid templates are [vanilla, react, vue3, vue2, lit, svelte, solid, preact, nestjs, tauri, tauri-React - (https://react.dev/), tauri-Vue3 - (https://vuejs.org/), tauri-Vanilla, tauri-Svelte - (https://svelte.dev/), tauri-Solid - (https://solidjs.com/), tauri-Preact - (https://preactjs.com/), electron, electron-React - (https://react.dev/), electron-Vue3 - (https://vuejs.org/), electron-Vanilla, electron-Svelte - (https://svelte.dev/), electron-Solid - (https://solidjs.com/), electron-Preact - (https://preactjs.com/)]

npm ERR! code 1
npm ERR! path /Users/joao.palmeiro/Documents
npm ERR! command failed
npm ERR! command sh -c create-farm my-vue-app --template vue

npm ERR! A complete log of this run can be found in: /Users/joao.palmeiro/.npm/_logs/2024-06-22T08_02_31_878Z-debug-0.log
```

I think it's because the `vue` template doesn't exist (the [Vue-related templates](https://github.com/farm-fe/farm/blob/v1.2.0/crates/create-farm-rs/src/template.rs#L160) are the `vue3` and `vue2` templates). Furthermore, these commands are not consistent with each other as Yarn's, for example, uses the `react` template. So, I think it would be useful to standardize the template chosen in these commands for `react` or `vue3` (and the project folder name if necessary).

Let me know your thoughts and if I can open a PR!

Thanks! 😄

--- a/docs/quick-start.mdx
+++ b/docs/quick-start.mdx
@@ -40,17 +40,17 @@ You can also directly specify the project name and the template you want to use 
 <>
     <Tabs>
           <TabItem value="npm 6.x" label="npm 6.x">
-              <CodeBlock>npm create farm@latest my-vue-app --template react</CodeBlock>
+              <CodeBlock>npm create farm@latest my-react-app --template react</CodeBlock>
               </TabItem>
 
   <TabItem value="npm 7+" label="npm 7+">
-          <CodeBlock>npm create farm@latest my-vue-app -- --template vue</CodeBlock>
+          <CodeBlock>npm create farm@latest my-react-app -- --template react</CodeBlock>
               </TabItem>
   <TabItem value="yarn" label="yarn">
-          <CodeBlock>yarn create farm my-vue-app --template react</CodeBlock>
+          <CodeBlock>yarn create farm my-react-app --template react</CodeBlock>
               </TabItem>
   <TabItem value="pnpm" label="pnpm">
-          <CodeBlock>pnpm create farm my-vue-app --template vue</CodeBlock>
+          <CodeBlock>pnpm create farm my-react-app --template react</CodeBlock>
               </TabItem>
           </Tabs>
 </>

--- a/i18n/zh/docusaurus-plugin-content-docs/current/quick-start.mdx
+++ b/i18n/zh/docusaurus-plugin-content-docs/current/quick-start.mdx
@@ -39,16 +39,16 @@ Farm 需要 **Node 16.18.0 及更高版本**。
 <>
     <Tabs>
           <TabItem value="npm 6.x" label="npm 6.x">
-              <CodeBlock>npm create farm@latest my-vue-app --template react</CodeBlock>
+              <CodeBlock>npm create farm@latest my-react-app --template react</CodeBlock>
               </TabItem>
   <TabItem value="npm 7+" label="npm 7+">
-          <CodeBlock>npm create farm@latest my-vue-app -- --template vue</CodeBlock>
+          <CodeBlock>npm create farm@latest my-react-app -- --template react</CodeBlock>
               </TabItem>
   <TabItem value="yarn" label="yarn">
-          <CodeBlock>yarn create farm my-vue-app --template react</CodeBlock>
+          <CodeBlock>yarn create farm my-react-app --template react</CodeBlock>
               </TabItem>
   <TabItem value="pnpm" label="pnpm">
-          <CodeBlock>pnpm create farm my-vue-app --template vue</CodeBlock>
+          <CodeBlock>pnpm create farm my-react-app --template react</CodeBlock>
               </TabItem>
           </Tabs>
 </>

